flowauth/backend/tests/test_token_admin.py

```diff
@@ -13,3 +16,20 @@ def test_list_tokens_for_server(client, auth, test_admin):
     response = client.get("/admin/tokens", headers={"X-CSRF-Token": csrf_cookie})
     assert 200 == response.status_code
     assert 1 == len(response.get_json())
+
+
+@pytest.mark.usefixtures("test_data")
+def test_get_server_public_key(app, client, auth):
+    """Test that admin can get the flowauth server's public key"""
```

Comment:
Should we add a test that regular users can't retrieve the public, or is this already covered somewhere?

Thinking about it, it actually shouldn't be a problem if they _can_ retrieve it, although there is no need to do so.